packaged/.prettierrc

```diff
@@ -9,5 +9,9 @@
       "files": "*.{cmp,page,component}",
       "options": { "parser": "html" }
     }
-  ]
+  ],
+  "useTabs": true,
```

Comment:
Well, by default I use space, not tabs in my projects, now once you change that rule, it brings a lot of white space changes